### ISSUE TITLE
Schematic and Endstop Pin Voltage

### ISSUE THREAD

**Issue by brotherdust** [Annex-Engineering/Constellation#5]
Hi!
This is a nice and simple board and I've had good luck with it. I wanted to try to make it so I can use a SuperPINDA as the endstop for my z-axis, so I flipped the related DIP switch and checked the voltage and polarity on the pins. It looks like the endstop pins only output 2.5v, which is lower than what I can power my PINDA with. I figure I can power it from elsewhere, so no big deal there. The question is how these endstop pins are wired. Since there's no schematic, I can't see where the power for the pins comes from nor the MCU pin it should be connected to. Presumably, the input pin is the same that was connected to the DIAG pin previously for sensorless homing.

Anyway, long story short, can you please provide schematics for this board? It would be very helpful.
Thanks!

**Comment by Kirinkunhime** [comment]
Hi !

I think you might have gotten a few things mixed up. The Dip switch does not need to be turned on when using an endstop - in your case you might have connected the Diag-pins of the Driver with the endstop (which is not how it's supposed to be used)

The Endstop pins go straight towards the DIAG1/2/3/4 pins on the Pico respectively.
The other pin is connected directly to ground.

As for the schematics, I've mentioned it quite often when poked in Discord but Supernova (as well as future constellation boards) will be fully closed source. This includes me not even sharing schematics.
While I personally don't like having to take this step, I kindly ask for your understanding, but that goes onto a bit of a longer rant. But basically, inside the DIY 3D printing community the somewhat recent culture has become to take something someone else made, spin it up slightly differently and call it your own thing. 
While there have been notable exceptions (shoutout to the peeps from the CroXY discord that e.g. credited the Carabiner as inspirations/similar designs), more often than not things will be _very closely inspired_ and if similarties are pointed out they're just ignored.

So once again I apologize for not providing schematics, but hope that I could help you with your issue regarding the SuperPINDA.

Also, after looking into the SuperPinda wiring really quick I'd not recommend using the SuperPinda with Supernova - this is due to the SuperPinda likely pulling up the sensor pin to 5V when it detects it's close enough. However, the Pico does not handle 5V on it's pins - it's limited to 3.3V.
But, with multi MCU homing it should be possible to connect the SuperPinda on another board (that you're likely running anyways for your heaters, as Supernova does not support it)

**Comment by brotherdust** [comment]
I appreciate your candor and completely understand your position. Sorry I missed the Discord thing; I am strongly against using Discord and its ilk as a "source of truth"/documentation for _any_ project, so tend to avoid using it. May I recommend that you update your _documentation_ here on the repo to reflect your position? As a courtesy and a benefit to the community, I'll gladly author a PR if you lack the time. Thanks much!

Edit: I forgot to address your recommendations: thank you for the heads up with the PINDA. I've applied a level-shifter to prevent this problem. I had been using multi-MCU before, but keep getting probe timeouts (it's an old einsy board), so wanted to get this as direct as possible.

**Comment by Kirinkunhime** [comment]
No problem, it is already mentioned in the readme (at least gerbers & source was mentioned there) so I've updated the readme to reflect that this does also include schematics.

I'll work on a more fleshed out answer (that does reflect the position of other people at Annex that are contributing to the boards as well) in the future :)
